### PR TITLE
INTYGFV-14455: Subscription warning modal

### PR DESCRIPTION
**Pull request by HelenaEdblad**

### CODE REVIEW

File: packages/webcert/src/feature/subscription/SubscriptionWarningModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import dispatchHelperMiddleware, { clearDispatchedActions } from '../../store/test/dispatchHelperMiddleware'
+import { createMemoryHistory } from 'history'
+import SubscriptionWarningModal from './SubscriptionWarningModal'
+import { updateUserResourceLinks } from '../../store/user/userActions'
+import { getSubscriptionWarningResourceLink } from '@frontend/common'
+import userEvent from '@testing-library/user-event'
+
+let testStore: EnhancedStore
+const history = createMemoryHistory()
+history.push = jest.fn()
```
Comment (perfli) [inline packages/webcert/src/feature/subscription/SubscriptionWarningModal.test.tsx:17]: Why are we mocking history.push() for these tests? Is it just copy/paste from another file?

File: packages/common/src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -29,18 +22,19 @@ export const Footer: React.FC = () => (
...
-                <ExternalLinkIconImg src={externalLinkIcon} />
+                <ExternalLinkIcon className="iu-ml-200 iu-fs-100" />
               </a>
             </li>
             <li>
               <a
                 className="ic-link-chevron ic-link ic-link--external"
                 target="_blank"
-                href="https://www.inera.se/kontakta-oss/felanmalan-och-anvandarstod/">
+                href="https://www.inera.se/kontakta-oss/felanmalan-och-anvandarstod/"
```
Comment (mhornfeldt) [inline packages/common/src/components/Footer/Footer.tsx:34]: Shouldn't the dynamic links be used instead?

File: packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      <p className="iu-mb-1rem">Du ser detta meddelande för att din verksamhet saknar abonnemang för Webcert.</p>
+
+      <p className="iu-mb-1rem">
+        För att er verksamhet fortsatt ska kunna utfärda intyg efter <span className="iu-color-error">2022-05-10</span> kräver Inera att ni
+        tecknar kundavtal samt beställer abonnemang för tjänsten Webcert.
+      </p>
+
+      <p className="iu-mb-1rem">
+        Om abonnemang saknas efter <span className="iu-color-error">2022-05-10</span> kommer inga nya intyg att kunna utfärdas. Er
+        verksamhet får då endast tillgång till Webcert för att läsa tidigare utfärdade intyg.
```
Comment (perfli) [inline packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx:20]: I think this text reads a little off. Are we sure this is how we want to phrase the information?

File: packages/webcert/src/store/user/userReducer.ts [hunk 4]
```diff
@@ -65,6 +66,9 @@ const userReducer = createReducer(initialState, (builder) =>
     .addCase(updateIsCareProviderModalOpen, (state, action) => {
       state.isCareProviderModalOpen = action.payload
     })
+    .addCase(removeResourceLink, (state, action) => {
+      state.links = state.links.filter((l) => l.type !== action.payload)
```
Comment (mhornfeldt) [inline packages/webcert/src/store/user/userReducer.ts:70]: I would have used link instead of l so it's easier to read

**Review by perfli — CHANGES_REQUESTED** [review]

**Review by mhornfeldt — COMMENTED** [review]
Nice :) Two small comments

File: packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx (hunk 3 above)
Comment (HelenaEdblad) [reply packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx:20]: I think this text will change when the requirements are set.

File: packages/webcert/src/feature/subscription/SubscriptionWarningModal.test.tsx (hunk 1 above)
Comment (HelenaEdblad) [reply packages/webcert/src/feature/subscription/SubscriptionWarningModal.test.tsx:17]: Yes, copy/paste :) I fix.

**Review by mhornfeldt — APPROVED** [review]

**Pull request merged**